src/LinuxTracingIntegrationTests/LinuxTracingIntegrationTest.cpp

```diff
@@ -328,7 +328,10 @@ class LinuxTracingIntegrationTestFixture {
  public:
   LinuxTracingIntegrationTestFixture() : puppet_{&LinuxTracingIntegrationTestPuppetMain} {}
 
-  [[nodiscard]] pid_t GetPuppetPid() const { return puppet_.GetChildPid(); }
+  [[nodiscard]] pid_t GetPuppetPidNative() const { return puppet_.GetChildPidNative(); }
+  [[nodiscard]] uint32_t GetPuppetPid() const {
+    return orbit_base::GetThreadIdFromNative(GetPuppetPidNative());
```

Comment:
Shouldn't this call `GetProcessIdFromNative`?

Reply:
Yes good point, thanks. Done